src/app/utils/keyboard.ts

```diff
@@ -11,23 +12,36 @@ export interface KeyboardEventLike {
   preventDefault(): void;
 }
 
+export function isComposing(evt: object): boolean {
+  if ('nativeEvent' in evt && typeof evt.nativeEvent === 'object' && evt.nativeEvent !== null) {
+    return isComposing(evt.nativeEvent)
+  }
+  if (isTaggedAsComposing(evt)) {
+    return true
+  }
+  if ('isComposing' in evt && typeof evt.isComposing === 'boolean') {
+    return evt.isComposing
+  }
+  return false
+}
+
 export const onTabPress = (evt: KeyboardEventLike, callback: () => void) => {
-  if (isKeyHotkey('tab', evt)) {
+  if (!isComposing(evt) && isKeyHotkey('tab', evt)) {
     evt.preventDefault();
     callback();
   }
 };
 
 export const preventScrollWithArrowKey: KeyboardEventHandler = (evt) => {
-  if (isKeyHotkey(['arrowup', 'arrowright', 'arrowdown', 'arrowleft'], evt)) {
+  if (!isComposing(evt.nativeEvent) && isKeyHotkey(['arrowup', 'arrowright', 'arrowdown', 'arrowleft'], evt)) {
```

Comment:
Is this change necessary? I've tested it with Chrome + Chinese IME and try to press arrow keys while composing, and it seems this change is not needed. Or maybe I missed something?